lib/usd/translators/meshWriter_BlendShapes.cpp

```diff
@@ -636,8 +674,13 @@ MObject PxrUsdTranslators_MeshWriter::writeBlendShapeData(UsdGeomMesh& primSchem
                         CHECK_MSTATUS_AND_RETURN(stat, MObject::kNullObj);
                         // NOTE: (yliangsiew) Because a single weight can drive multiple targets, we
                         // have to put a numeric suffix in the target name.
```

Comment:
Is this comment still accurate now? Or maybe it's only accurate in a certain condition (`k != 0`)?

Reply:
Resolved in 522b6b0.